examples/spark/README.md

```diff
@@ -149,66 +181,73 @@ $ kubectl logs spark-master
 The Spark driver is used to launch jobs into Spark cluster. You can read more about it in
 [Spark architecture](http://spark.apache.org/docs/latest/cluster-overview.html).
 
-```shell
-$ kubectl create -f examples/spark/spark-driver.json
+```console
+$ kubectl create -f examples/spark/spark-driver-controller.yaml
+replicationcontrollers/spark-driver-controller
 ```
 
 The Spark driver needs the Master service to be running.
 
 ### Check to see if the driver is running
 
-```shell
-$ kubectl get pods
-NAME                                           READY     REASON    RESTARTS   AGE
-[...]
-spark-master                                    1/1       Running   0          14m
-spark-driver                                    1/1       Running   0          10m
+```console
+$ kubectl get pods -lcomponent=spark-driver
+NAME                            READY     STATUS    RESTARTS   AGE
+spark-driver-controller-vwb9c   1/1       Running   0          1m
 ```
 
 ## Step Four: Do something with the cluster
 
-Use the kubectl exec to connect to Spark driver
+Use the kubectl exec to connect to Spark driver and run a pipeline.
 
-```
-$ kubectl exec spark-driver -it bash
-root@spark-driver:/#
-root@spark-driver:/# pyspark
+```console
+$ kubectl exec spark-driver-controller-vwb9c -it pyspark
 Python 2.7.9 (default, Mar  1 2015, 12:57:24)
 [GCC 4.9.2] on linux2
 Type "help", "copyright", "credits" or "license" for more information.
-15/06/26 14:25:28 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
+15/10/26 21:03:17 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
```

Comment:
Looks fixable (in the next image)

http://stackoverflow.com/questions/19943766/hadoop-unable-to-load-native-hadoop-library-for-your-platform-warning